src/brasil/gov/vlibrasnews/tests/test_subscribers.py

```diff
@@ -40,23 +41,23 @@ def test_validate(self):
 
     def test_create_translation_ok(self):
         with HTTMock(vlibras_ok):
-            self.assertTrue(subscribers.create_translation(self.document, None))
+            self.assertTrue(subscribers.create_translation(self.document, Mock(action='publish')))
```

Comment:
this is not how a subscriber must be tested; you should fire the event instead.

for some code examples see: https://github.com/collective/collective.fingerpointing/blob/master/src/collective/fingerpointing/tests/test_pas_subscriber.py